wildfly-getting-started-archetype/src/main/resources-filtered/archetype-resources/pom.xml

```diff
@@ -51,8 +51,27 @@
         <maven.compiler.release>${maven.compiler.release}</maven.compiler.release>
     </properties>
 
-    <!-- the JBoss community and Red Hat GA Maven repositories -->
+    <!--
+    Repositories are defined in the order that they should be used.
+    (1) Maven central, (2) JBoss community, (3) Red Hat GA Maven repository
```

Comment:
@soul2zimate Is that comment right? We don't need Red Hat GA Maven repository for WildFly archetypes as far as I know

Reply:
@jmesnil you're correct, I removed the 3rd one from the comment.